pkg/mutation/mutators/core/errors.go

```diff
@@ -4,4 +4,7 @@ import "errors"
 
 // ErrNonKeyedSetter occurs when a setter that doesn't understand keyed lists
 // is called against a keyed list.
-var ErrNonKeyedSetter = errors.New("mutator does not understand keyed lists")
+var (
+	ErrNonKeyedSetter = errors.New("mutator does not understand keyed lists")
+	ErrNameLength     = errors.New("maximum length for name is 63 char")
```

Comment:
"char" => "characters", this may be user-visible.

Also "maximum name length is 63 characters" is slightly cleaner phrasing.